src/test/java/suite/operator/V2ApiOperatorLocalOnlyTest.java

```diff
@@ -0,0 +1,25 @@
+package suite.operator;
+
+
+import app.component.Operator;
+import com.uid2.client.DecryptionResponse;
+import com.uid2.client.IdentityTokens;
+import com.uid2.client.TokenGenerateResponse;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SuppressWarnings("unused")
+public class V2ApiOperatorLocalOnlyTest {
```

Comment:
Why is this own separate suite?

Reply:
I've consolidated it into V2ApiOperatorTest